packratt/registry.py

```diff
@@ -20,13 +21,20 @@
 
 del SCHEMA_PATH
 
+# user defined registry
+USER_REGISTRY = Path(user_config_dir, "registry.yaml")
+
 
 def load_registry(filename=None):
 
     if filename is None:
+
         path = Path(Path(packratt.__file__).parent,
                     "conf", "registry.yaml")
 
+        if USER_REGISTRY.is_file():
+            path = USER_REGISTRY
+
```

Comment:
Applications frequently support a hierarchy of configurations, from global all the way to user configurations.

I think that the user registry should augment the global registry, not replace it.

I'd remove the above three lines.